azurerm/mssql-database/variables.tf

```diff
@@ -10,9 +10,13 @@ variable "name_override" {
   default     = ""
 
   validation {
-    condition     = var.name_override == "" || (can(regex("^[^<>*%&:\\/?]+$", var.name_override)) && !can(regex("[. ]$", var.name_override)) && length(var.name_override) <= 128 && length(var.name_override) >= 1)
+    condition     = var.name_override == "" || (can(regex("^[^<>*%&:\\/?]+$", var.name_override)) && !can(regex("[. ]$", var.name_override)))
     error_message = "Invalid name_override. Name must not contain any of the following characters: <, >, *, %, &, :, \\, /, ?, and it must not end with a period or space. Name must be between 1 and 128 characters long."
```

Comment:
Fjerne. Nevnes i linje 18 og denne valideringen sjekker ikke lengde

Reply:
"Name must be between 1 and 128 characters long."

Reply:
Fixed 👌